nVolve/Features/Content/ContentViewModel.swift

```diff
@@ -9,13 +9,12 @@ import Alamofire
 import MapKit
 import SwiftUI
 
-@Observable class ContentViewModel {
-    var position: MapCameraPosition = .automatic
-    var showEventInfo: Bool = false
-    var showingFilters: Bool = false
-    var events: [InvolvedEvent] = []
-    // ':' is encoded as %3
-    // 'T' is the separator between time and date
+class ContentViewModel: ObservableObject {
```

Comment:
https://developer.apple.com/documentation/swiftui/migrating-from-the-observable-object-protocol-to-the-observable-macro